omnipaxos_core/src/util.rs

```diff
@@ -45,6 +45,8 @@ where
 {
     pub n_leader: Ballot,
     pub promises_meta: Vec<Option<PromiseMetaData>>,
+    // the sequence number of accepts for each follower where AcceptSync has sequence number = 1
+    pub accept_sequences: Vec<u64>,
```

Comment:
Same here with naming:

```suggestion
    pub follower_seq_nums: Vec<u64>,
```